prover/vk_setup_data_generator_server_fri/src/keystore.rs

```diff
@@ -316,37 +301,45 @@ impl Keystore {
     /// Keys are loaded from the default 'base path' files.
     pub fn load_keys_to_data_source(&self) -> anyhow::Result<InMemoryDataSource> {
         let mut data_source = InMemoryDataSource::new();
-        for base_circuit_type in
-            (BaseLayerCircuitType::VM as u8)..=(BaseLayerCircuitType::L1MessagesHasher as u8)
+        for base_circuit_type in (BaseLayerCircuitType::VM as u8
+            ..=BaseLayerCircuitType::Secp256r1Verify as u8)
+            .chain(once(BaseLayerCircuitType::EIP4844Repack as u8))
         {
             data_source
                 .set_base_layer_vk(self.load_base_layer_verification_key(base_circuit_type)?)
                 .unwrap();
         }
 
-        for circuit_type in ZkSyncRecursionLayerStorageType::SchedulerCircuit as u8
-            ..=ZkSyncRecursionLayerStorageType::LeafLayerCircuitForL1MessagesHasher as u8
+        for circuit_type in (ZkSyncRecursionLayerStorageType::SchedulerCircuit as u8
+            ..=ZkSyncRecursionLayerStorageType::LeafLayerCircuitForEIP4844Repack as u8)
+            .chain(once(
+                ZkSyncRecursionLayerStorageType::RecursionTipCircuit as u8,
+            ))
```

Comment:
related to the other comment on this - this seems to be equivalent to ` (ZkSyncRecursionLayerStorageType::SchedulerCircuit as u8
            ..=ZkSyncRecursionLayerStorageType:: RecursionTipCircuit as u8)` or just `ZkSyncRecursionLayerStorageType::all()`